configure.ac

```diff
@@ -234,13 +236,22 @@ OMPI_CHECK_PORTALS4([portals4],
     [transport_portals4="no"])
 AM_CONDITIONAL([USE_PORTALS4], [test "$transport_portals4" = "yes"])
 
-OMPI_CHECK_OFI([ofi],
+OPAL_CHECK_LIBFABRIC([libfabric],
     [AC_DEFINE([USE_OFI], [1], [Define if OFI transport active])
      transport="ofi"
      transport_ofi="yes"],
     [transport_ofi="no"])
 AM_CONDITIONAL([USE_OFI], [test "$transport_ofi" = "yes"])
 
+if test "$transport_ofi" = "no" ; then
+  OMPI_CHECK_OFI([ofi],
```

Comment:
Oh, ick.  It would be nice to get rid of this old macro.  Could you modify the ```AC_HELP_STRING``` in ```ompi_check_ofi.m4``` and prefix the message with "(deprecated)"?